src/main/java/net/foxes4life/foxclient/screen/settings/FoxClientSettingsScreen.java

```diff
@@ -39,7 +39,7 @@ public class FoxClientSettingsScreen extends Screen {
     static double entryHoverBgY = 0;
     static int entryHoverBgYGoal = 0;
 
-    private static final int sidebarWidth = 64 + 16;
+    private static final int sidebarWidth = 64 + 32;
```

Comment:
i get that this is needed for longer texts but i think it might be smarter to iter through the text lengths and then determine if this should be increased
for now this is sufficient though

Reply:
i have no clue, how to do that, so i'll leave that up to you and flustix